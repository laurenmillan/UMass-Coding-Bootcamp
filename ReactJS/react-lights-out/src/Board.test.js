import React from 'react';
import { render } from '@testing-library/react';
import Board from './Board';

// SMOKE TEST
describe('Board', () => {
	test('it renders without crashing', () => {
		render(<Board />);
	});
});

// SNAPSHOT TEST
test('it matches snapshot', () => {
	const { asFragment } = render(<Board />);
	expect(asFragment()).toMatchSnapshot();
});
