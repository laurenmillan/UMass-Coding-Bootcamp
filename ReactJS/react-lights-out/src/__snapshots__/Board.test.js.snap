// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Board matches snapshot 1`] = `
<DocumentFragment>
  <table
    class="Board"
  >
    <tbody>
      <tr>
        <td
          class="Cell Cell-lit"
        />
        <td
          class="Cell Cell-lit"
        />
        <td
          class="Cell "
        />
        <td
          class="Cell "
        />
      </tr>
      <tr>
        <td
          class="Cell "
        />
        <td
          class="Cell "
        />
        <td
          class="Cell "
        />
        <td
          class="Cell "
        />
      </tr>
      <tr>
        <td
          class="Cell "
        />
        <td
          class="Cell "
        />
        <td
          class="Cell "
        />
        <td
          class="Cell Cell-lit"
        />
      </tr>
      <tr>
        <td
          class="Cell "
        />
        <td
          class="Cell "
        />
        <td
          class="Cell Cell-lit"
        />
        <td
          class="Cell "
        />
      </tr>
    </tbody>
  </table>
</DocumentFragment>
`;

exports[`it matches snapshot 1`] = `<DocumentFragment />`;
